packages/svelte/tests/runtime-legacy/samples/await-mutate-array/main.svelte

```diff
@@ -0,0 +1,34 @@
+<script>
+	import Card from './Card.svelte'
+
+	const x = [{
+		fav: false,
+		x: 1
+	}, {
+		fav: false,
+		x: 2
+	}, {
+		fav: false,
+		x: 3
+	}, {
+		fav: false,
+		x: 4
+	}];
+
+	let p_cards = Promise.resolve(JSON.parse(JSON.stringify(x)));
+
+	let show = true;
```

Comment:
```suggestion
```

Reply:
I'm sorry I messed up the last code suggestion, should be fine now. Also is there anyway this could've affected areas other than awaited objects?

Reply:
If they use await blocks in non-runes mode, yes.